CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-Below is a non-exhaustive list of changes between `gen_rpc` versions.
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
^ Should it be mentioned that the project follows Semantic Versioning _starting_ at whatever version it did? (Perhaps the next one to be released?)

I.e. "This project adheres to [Semantic Versioning](...) starting at version X.Y.Z"

Reply:
Nice catch. Not sure about the past, but we can play on the safe side.

Reply:
6aee28d

Reply:
👍